package.json

```diff
@@ -50,9 +50,13 @@
     "jsdom": "^9.5.0",
     "sinon": "^1.17.6",
     "tslint": "^5.2.0",
-    "typescript": "~2.4.1"
+    "typescript": "~2.5.2"
   },
   "dependencies": {
+    "@dojo/core": "next",
```

Comment:
Did you mean to add these as deps?

Reply:
DOH! no... it was the freaking lack of `--no-save` and then me thinking it was just overwriting what was already there...

Reply:
The next release of grunt-dojo2 *should* fix that 😄